src/hooks/useI18n.ts

```diff
@@ -11,7 +11,7 @@ const useI18n = () => {
 
   return (translationId: number, fallback: string, data: ContextData = {}) => {
     const foundTranslation = translations.find((translation) => {
-      return translation.data.stringId === translationId
+      return translation.data?.stringId === translationId
```

Comment:
Can you explain why this change ?

Reply:
avoid app crash if translations cant get load